scrapesymbols/gathersymbols.py

```diff
@@ -184,16 +153,11 @@ def main():
     if not os.path.isabs(args.dump_syms) or not os.path.exists(args.dump_syms) or not os.access(args.dump_syms, os.X_OK):
         print >>sys.stderr, 'Error: can\'t find dump_syms binary at %s!' % args.dump_syms
         return 1
-    if args.all or args.files:
-        missing_symbols = None
```

Comment:
Wasn't the script already doing what you describe by setting missing_symbols to None when args.all was True?

Reply:
I'm trying to get rid of the functionality that queries a server for missing symbols. As best I can tell it's redundant. I'll say more below.